src/main/java/com/iotiq/application/domain/Product.java

```diff
@@ -60,8 +64,7 @@ public class Product extends AbstractPersistable<UUID> {
     private ApplicationAreaGroup applicationAreaGroup;
 
     //Marken
-    @Enumerated(EnumType.STRING)
-    private Brand brand;
+    private String brand;
```

Comment:
If the products can have the same brands maybe brand can be an entity. @oguzIotiq

Reply:
We decided to have the brands as a string field in products since, for now, brands won't have a property other than the name